sdk/js/src/svm.ts

```diff
@@ -161,34 +161,22 @@ export async function constructSwapInstruction(
     router,
   } = extractSwapInfo(swapOpportunity);
 
-  if (
-    swapOpportunity.tokens.type === "searcher_specified" &&
-    swapOpportunity.feeToken === "user_token"
-  ) {
-    // scale bid amount by FEE_SPLIT_PRECISION/(FEE_SPLIT_PRECISION-fees) to account for fees
-    const denominator = FEE_SPLIT_PRECISION.sub(
-      new anchor.BN(
-        swapOpportunity.platformFeeBps + swapOpportunity.referralFeeBps,
-      ),
-    );
-    const numerator = bidAmount.mul(FEE_SPLIT_PRECISION);
-    // add denominator - 1 to round up
-    bidAmount = numerator
-      .add(denominator.sub(new anchor.BN(1)))
-      .div(denominator);
-  }
+  const bidAmountIncludingFees = getBidAmountIncludingFees(
```

Comment:
I renamed the variable because I don't like mutating the argument `bidAmount` like we did previously